WalletWasabi/GitInfo/GitInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.IO;
```

Comment:
```suggestion
#nullable enable

﻿using System.IO;
```

Reply:
This should be used in new code. Once we decide strategy how to migrate to nullable reference types, we can either remove those directives or let them be. In both cases, it's good to write it in new code.

Reply:
Not needed for now, i can return N/A string in the GitInfo class itself

Reply:
You can but then you are mixing the logic for getting information (business logic) with presentation information. So that when somebody else would want to use `GitInfo`, she would have to refactor to that.